pkgs/by-name/kc/kcc/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  qt6,
+  python3Packages,
+  fetchFromGitHub,
+  p7zip,
+  archiveSupport ? true,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "kcc";
+  version = "6.2.2";
+
+  src = fetchFromGitHub {
+    owner = "ciromattia";
+    repo = "kcc";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GX0WPu1EZx/8H1ajivxgnym4jECmwPBJU25vzcfOQCw=";
+  };
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook ];
+
+  buildInputs = lib.optional stdenv.hostPlatform.isLinux qt6.qtwayland;
```

I think `lib.optionals` is preferred over `lib.optional`.

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ qt6.qtwayland ];
```